### PR TITLE
253/add story four

### LINKED ISSUE #253: Allow for different number of stories
## Description
Story creation/rendering should be done dynamically. Some projects come with 2 stories, others with 3 or 4. We need to come up with a way to create a template class and be able to create as many stories as needed for each project

## Acceptance Criteria
- [ ] Some brands have 4 stories (e.g. seap)
- [ ]  Others do not (e.g. Hospice Hope)

## Plan
- [x] Refactor/rework the StoryDeck.elm - the decks part

## Previous attempts
-> #108 - closed

### PR DESCRIPTION
**Pull request by katjam**
Fixes #253 

## Description

- Added a forth story with optional display

@neontribe/contemplating-action

### CODE REVIEW

File: src/StoryDeck.elm [hunk 1]
```diff
@@ -287,4 +300,33 @@ decks =
               }
             ]
       }
+    , { id = 4
+      , title = StoryFourTitle
+      , teaser = StoryFourTeaser
+      , teaserImgPath = AssetPath "story_images/story4-thumbnail.jpg"
+      , teaserImgAltText = StoryFourTeaserImageAlt
+      , relatedInfo = relatedInfo 3
```
Comment (AlinChindea) [inline src/StoryDeck.elm]: should this be 3 or 4? the story with id = 3 has `relatedInfo 3` too.

**Review by AlinChindea — COMMENTED** [review]
for story 4, the `relatedInfo` key has a value of `relatedInfo 3` but maybe it should be 4. :question: :thinking:

File: src/StoryDeck.elm (hunk 1 above)
Comment (katjam) [reply src/StoryDeck.elm]: Nice.

**Review by AlinChindea — APPROVED** [review]
:film_projector: let's keep rolling! :rocket:

**Pull request merged**